prov/verbs/src/fi_verbs.h

```diff
@@ -186,6 +207,14 @@ struct fi_ibv_msg_ep {
 	struct fi_ibv_cq	*scq;
 	uint64_t		ep_flags;
 	struct fi_info		*info;
+	atomic_t		unsignaled_send_cnt;
+	atomic_t		comp_pending;
+	uint64_t		ep_id;
+};
+
+struct fi_ibv_msg_epe {
+	struct slist_entry	entry;
+	struct fi_ibv_msg_ep 	*ep;
```

Comment:
The ep_list uses a custom matching function when searching through the list. So fid_list cannot be used here.